fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LucenOnlineIndexingTest.java

```diff
@@ -937,5 +939,70 @@ void testRecordUpdateReducedMergeForcingAgileTimeQuota() {
         assertTrue(newLength < oldLength);
         assertTrue(timeCommitCount > 0);
     }
+
+    private static class TerribleRebanalceIndexMaintainer extends LuceneIndexMaintainer {
```

Comment:
```suggestion
    private static class TerribleRebalanceIndexMaintainer extends LuceneIndexMaintainer {
```

I'm also not sure that putting `Terrible` in here is super valuable, but it's not a big deal.

Reply:
✅

Reply:
(The "Terrible" seems to be an ad-hoc description of a test maintainer that doesn't do its job...)